### PR TITLE
[REM] *: Remove non-migrated modules and prepare branch for v13

### PR DESCRIPTION
**Pull request by luisg123v**
This removes 13 modules that were present but not actually migrated
yet.

In addition, this changes version in the following places:
- Module `account_multicompany_code` (the only actually migrated)
- The `.travis.yml` file
- The CONTRIBUTING file

### CODE REVIEW

**Comment by luisg123v** [comment]
@hugho-ad @moylop260 @suniagajose 

Could you review, please?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ How add a new feature:
 12. Push your first change branch to know you start to work on.
 
     ```bash
-    $ git push -u vauxoo-dev 12.0-your_new_feature_theme
+    $ git push -u vauxoo-dev 1230-your_new_feature_theme
```
Comment (suniagajose) [inline CONTRIBUTING.md]: hi @luisg123v 

there is a typo error?

regards,

**Review by suniagajose — APPROVED** [review]

**Comment by luisg123v** [comment]
Thanks a lot @suniagajose, fixed.

I also removed server-tools from the CONTRIBUTING file and fixed module lints on a separated commit.

**Comment by moylop260** [comment]
@hugho-ad 

Could you check it, please?

File: .travis.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ cache:
     - $HOME/.cache/pip
 
 addons:
-  postgresql: "9.6"
+  postgresql: "10"
```
Comment (hugho-ad) [inline .travis.yml:10]: why not 11?

Comment (luisg123v) [reply .travis.yml:10]: I first tried 11, but, unfortunately, it didn't work because that version is not preinstalled on travis yet.

**Comment by hugho-ad** [comment]
LGTM :+1:

**Pull request merged**